### DISCUSSION TITLE
How to match a given RGB colour?

### DISCUSSION THREAD

**Discussion by babyta** [colour-science/colour#1256] [Q&A]
### Question

My current question is if I input a color RGB, how do I output a color that matches it? This matching color matches the input color and looks coordinated with each other.

**Comment by thomasmansencal** [comment]
Hi @babyta,

I'm not sure to understand the question fully, would it be possible to add more details please?

**Comment by babyta** [comment]
For example, if I want to design a flat picture for e-commerce, for a given product, I can get the color of the product through clustering. So what colors should I use for elements such as text and color blocks? I want to get a new color or color system based on the product color. This new color matches the product.

**Reply by thomasmansencal** [reply]
Oh you mean doing something akin to palette extraction?

**Reply by babyta** [reply]
That’s almost the meaning, but I need to match 2-3 colors which is enough

**Reply by thomasmansencal** [reply]
We do not really have any function to do that unfortunately, this is not really what colour is built for, that being said, there should be mostly everything needed to do that with a bit of additional code.

Were you thinking about something like that: https://color.adobe.com/create/color-wheel?

**Comment by babyta** [comment]
I'm not sure if the color library has ready-made functions

**Comment by babyta** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
